ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+### Problem:
```

Comment:
As food for thought I quite like the way eslint document their bug reporting requirements: http://eslint.org/docs/developer-guide/contributing/reporting-bugs